Tests/test_dynamic_loading.py

```diff
@@ -82,6 +82,11 @@ def test_can_import_trigger(self):
         triggers = self.get_package_in_folder(self.trigger_dir)
         package_name = "trigger"
         for trigger in triggers:
+
+            # snowboy is only compiled for python2
```

Comment:
It is linked against python2 symbols and has to be recompiled for python3 manually.